commands.js

```diff
@@ -59,20 +53,46 @@ function createCommandChoices() {
     return commandChoices;
 }
 
+// Get the challenge categories from api/leetcode-data.js
+export async function challengeCategoriesCommandChoices() {
+    const commandChoices = [];
+    for (const [key, value] of Object.entries(await getProblemCategories())) {
+        commandChoices.push({
+            name: key,
+            value,
+        });
+    }
```

Comment:
you could do this mapping in one line 
```js
const commandChoices = Object.entries(await getProblemCategories()).map(([key, value]) => ({
    name: key,
    value,
}));

```
or better yet, you could probably do `new Map(categories)`